src/dhtproto/client/legacy/internal/helper/RetryHandshake.d

```diff
@@ -118,10 +86,11 @@ class RetryHandshake
 
     /***************************************************************************
 
-        try doing the handshake
+        Try doing the handshake.
```

Comment:
Minor style: I thought we didn't have `.` at the end of the 'title' line of a ddoc block?

Reply:
Hmm, I always add one. And it's purely a coincidence that this comment is just one line long.

Reply:
That's weird, I have had the impression for a long time that the absence of one was actually a style requirement :-P

Reply:
I've never heard of it, if it is! If you can find a ref to this in our style guide, I'm happy to rethink my existence!

Reply:
Well, the style guide has a `.`, so maybe I should rethink mine! ;-)